README.md

```diff
@@ -7,6 +7,7 @@ Operator that deploying and controlling additional [KubeVirt](https://kubevirt.i
 - [VM Console Proxy](https://github.com/kubevirt/vm-console-proxy)
 - [Template Validator](https://github.com/kubevirt/ssp-operator/tree/main/internal/template-validator)
 - Metrics Rules (Currently there is just a single Prometheus rule that counts the number of running Virtual Machines.)
+- Virtual Machine Delete Protection
```

Comment:
Can you please add more details here? IMO, something like this:
```
- Virtual Machine (VM) Delete Protection (deploys ValidatingAdmissionPolicy (VAP) that prevents VM from being deleted when protection is enabled.)
```

Reply:
Done.

Reply:
@jcanocan Why you have 2 lines in this PR?

Reply:
Ups, my bad. Copy&paste mistake :sweat_smile: 
Dropped.